packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: For Developers
+order: 10
+---
+
+import { Alert } from '@cmsgov/design-system';
+
+Greetings, Developers! This page will walk you through all the basics of using the design system, but you might want to take a look at [our example projects on GitHub](https://github.com/CMSgov/design-system/tree/main/examples/) as well.
```

Comment:
Should we say why ppl would want to take a look at example projects Something like,
> ..., but you might want to take a look at our example projects on GitHub to see working examples of how the design system works in different builds.

Reply:
How about "but another good way to get a jump start is to look through [our example projects on GitHub](https://github.com/CMSgov/design-system/tree/main/examples/)"?